ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -152,8 +152,6 @@ function integration::bazel_with_emulators() {
     "generator/..."
     # BigQuery integration tests
     "google/cloud/bigquery/..."
-    # IAM and IAM Credentials integration tests
-    "google/cloud/iam/..."
```

Comment:
Hmm... With this change, we no longer run the samples in `google/cloud/iam/samples`. (yes the samples are a source of flakes (#7330, #9715)). But the Cloud C++ Snippets mini design doc says: "We wanted the snippets to be compiled and executed as part of the CI build".

Do we think compiling only is sufficient, now?

Reply:
No, I forgot about the examples. Restored.